src/firejail/main.c

```diff
@@ -1515,7 +1521,7 @@ int main(int argc, char **argv, char **envp) {
 		else if (strncmp(argv[i], "--cgroup=", 9) == 0) {
 			if (checkcfg(CFG_CGROUP)) {
 				if (option_cgroup) {
-					fprintf(stderr, "Error: only a cgroup can be defined\n");
+					fprintf(stderr, "Error: Only a cgroup can be defined\n");
```

Comment:
In this case the sentence isn't finished after the `:`, so no capital O here. And use a dot (.) to finish the sentence:
fprintf(stderr, "Error: only a cgroup can be defined.\n");
The final nitpick!

Reply:
oh, this is not connected by my pr and editing will be crash another test too.

Reply:
hmm. no typo in here, at least in my fork.

https://github.com/kuesji/firejail/blob/b8abb6ec67cdcd32e66990a7fc16fb5b29f9b9f3/src/firejail/main.c#L1524

Reply:
I do see a capital O in your fork link. After closer investigation of this file it seems nobody used a final dot so for consistency you can forget that comment:
 fprintf(stderr, "Error: only a cgroup can be defined\n");

Reply:
Revert back to https://github.com/netblue30/firejail/pull/4315/files#diff-6698244f9a67e5c8ae5c03806df74f6d9f1ae1b31ad6176eb09e136f07f3dad9L1518.